tests/system/test_system_async.py

```diff
@@ -339,6 +341,47 @@ async def test_document_update_w_int_field(client, cleanup, database):
     assert snapshot1.to_dict() == expected
 
 
+@pytest.mark.skipif(FIRESTORE_EMULATOR, reason="Require index and seed data")
```

Comment:
Are you sure this is required? Are there no other system tests that would require an index?

Reply:
Yeah this is the only one I'm aware of. All others are depending on built-in index (no need to bootstrap).